Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/FeatureView.cs

```diff
@@ -13,17 +16,52 @@ public class FeatureView : MonoBehaviour
         private GameObject[] _stageInteracts;
         private GameObject[] nearStageObject = null;
         private GameObject interactTarget = null;
+        private GameObject _floor;
         AudioSource _audioSource;
         private Subject<Unit> _onDestroy = new Subject<Unit>();
         public IObservable<Unit> onDestroy { get => _onDestroy; }
+        private ReactiveProperty<GameObject> floor = new ReactiveProperty<GameObject>();
+        public IObservable<GameObject> OnFloor { get => floor; }
+        private Vector3 direction = new Vector3(0, -1, 0);
+        private StageTile _stagetile;
+        public StageTile stagetile { get { return _stagetile; } }
```

Comment:
```suggestion
        private ReactiveProperty<StageTile> floor = new ReactiveProperty<StageTile>();
        public IObservable<StageTile> OnFloor { get => floor; }
        private Vector3 direction = new Vector3(0, -1, 0);
```
今思えばGameObjectにする必要がないので、クラスのまま渡していい気がしました